docs/book/src/reference/platform.md

```diff
@@ -0,0 +1,240 @@
+work no matter the operating system and architecture of the Kubernetes cluster where it is installed .
+
+## How to define which platforms are supported
+
+The following covers what you need to do to provide the support for one or more platforms or architectures.
+
+### 1) Build workload images to provide the support for other platform(s)
+
+The images used in workloads such as in your Pods/Deployments will need to provide the support for this other platform. 
+You can inspect the images using aManifestList of supported platforms using the command 
+[docker manifest inspect <image>][docker-manifest], i.e.:
+
+```shell
+$ docker manifest inspect myresgystry/example/myimage:v0.0.1
+{
+   "schemaVersion": 2,
+   "mediaType": "application/vnd.docker.distribution.manifest.list.v2+json",
+   "manifests": [
+      {
+         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
+         "size": 739,
+         "digest": "sha256:a274a1a2af811a1daf3fd6b48ff3d08feb757c2c3f3e98c59c7f85e550a99a32",
+         "platform": {
+            "architecture": "arm64",
+            "os": "linux"
+         }
+      },
+      {
+         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
+         "size": 739,
+         "digest": "sha256:d801c41875f12ffd8211fffef2b3a3d1a301d99f149488d31f245676fa8bc5d9",
+         "platform": {
+            "architecture": "amd64",
+            "os": "linux"
+         }
+      },
+      {
+         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
+         "size": 739,
+         "digest": "sha256:f4423c8667edb5372fb0eafb6ec599bae8212e75b87f67da3286f0291b4c8732",
+         "platform": {
+            "architecture": "s390x",
+            "os": "linux"
+         }
+      },
+      {
+         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
+         "size": 739,
+         "digest": "sha256:621288f6573c012d7cf6642f6d9ab20dbaa35de3be6ac2c7a718257ec3aff333",
+         "platform": {
+            "architecture": "ppc64le",
+            "os": "linux"
+         }
+      },
+   ]
+}
+```
+
+### 2) (Recommended as a Best Practice) Ensure that node affinity expressions are set to match the supported platforms
+
+Kubernetes provides a mechanism called [nodeAffinity][node-affinity] which can be used to limit the possible node 
+targets where a pod can be scheduled. This is especially important to ensure correct scheduling behavior in clusters 
+with nodes that span across multiple platforms (i.e. heterogeneous clusters).
+
+**Kubernetes manifest example**
+
+```yaml
+affinity:
+  nodeAffinity:
+    requiredDuringSchedulingIgnoredDuringExecution:
+      nodeSelectorTerms:
+      - matchExpressions:
+        - key: kubernetes.io/arch
+          operator: In
+          values:
+          - amd64
+          - arm64
+          - ppc64le
+          - s390x
+      - matchExpressions:
+          - key: kubernetes.io/os
```

Comment:
Shall we move this `kubernetes.io/os` stanza as a part of the `matchExpressions` above? That way it's for specifying the `kubernetes.io/arch` && `kubernetes.io/os` as linux rather than ||?

Reply:
done